apps/user/views.py

```diff
@@ -82,9 +82,9 @@ class ProfileChart(APIView):
   def get(self, request, exam_id):
     """Sends attempts data to frontend for charts."""
     show_entries = 50
-    attempts = ExamAttempt.objects.filter(exam=exam_id,
-                                          user_id=request.user.id,
-                                          mode='exam')[:show_entries]
+    attempts = ExamAttempt.objects.filter(
```

Comment:
Doesn't it need to be ordered  by `-created`?

Reply:
These are specific attempts in chart for each exam. With regular query they are sorted by created (asc). `-created` will make them look backwards. 
Also, should I specifically say order_by('created') anyway? Just in case it is taken from db in different order?